src/vm/dllimport.cpp

```diff
@@ -6507,6 +6573,40 @@ NATIVE_LIBRARY_HANDLE NDirect::LoadLibraryModuleBySearch(NDirectMethodDesc * pMD
     return hmod;
 }
 
+// Search for the library and variants of its name in probing directories.
+// This method returns native system libray handle not registered with the PAL
+// static
+NATIVE_LIBRARY_HANDLE NDirect::LoadLibraryModuleBySearch(NDirectMethodDesc * pMD, LoadLibErrorTracker * pErrorTracker, const wchar_t* wszLibName)
```

Comment:
I was under the impression that `wchar_t` should not be used in the code base and we should always use the `WCHAR` type?

Reply:
`wchar_t` is not strictly required - we `#define` it in the PAL to be `WCHAR`.

In this case, it would be better to use `PCWSTR` to make it clear that it is a zero terminated string.